### ISSUE TITLE
Support adding new connector to default tenant

### ISSUE THREAD

**Issue by ruckc** [FusionAuth/fusionauth-issues#982]
### Problem
We are unable to use kickstart to automatically configure a kickstart created LDAP Connector in the default tenant.  The tenant API requires updating the `tenant.connectorPolicies` with the array of all connectors, given the default FusionAuth connector doesn't have an accessible/pre-assigned UUID, we are unable to update the default tenant with the LDAP Connector.

### Solution
Expose a `ENV.DEFAULT_CONNECTOR_ID` like you already do for default tenant and the fusionauth application.

### Alternatives/workarounds
Currently the only method we can figure out is manually updating the default tenant via the admin web UI.

### How to vote
Please give us a thumbs up or thumbs down as a reaction to help us prioritize this feature. Feel free to comment if you have a particular need or comment on how this feature should work.

**Comment by mooreds** [comment]
@ruckc the fusionauth connector has a constant UUID. It's documented here:

https://fusionauth.io/docs/v1/tech/connectors/fusionauth-connector/

and is `e3306678-a53a-4964-9040-1c96f36dda72`. That may be a viable workaround for your situation.

I agree, though, it'd be more elegant to have it as an environment variable.

**Comment by ruckc** [comment]
So we tried this and multiple issues appeared with this kickstart request:

```json
          {
            "method": "PATCH",
            "url": "/api/tenant/{{ .Values.auth.defaultTenantId }}",
            "body": {
              "tenant": {
                "connectorPolicies": [
                  {"connectorId": "#{ldapConnector}" },
                  {"connectorId": "#{fusionauthConnector}" }
                ]
              }
            }
          },
```

1. Kickstart for Reactor License #949 
2. Looks like domains does not default to ["*"].  Adding `domains: ["*"]` removes this error. 
3. It is trying to merge all the connector polices, instead of overwriting the list.  Adding only the ldapConnector removes this error.

```
{
  "tenant" : {
    "connectorPolicies" : [ {
      "connectorId" : "005436a1-8f29-4642-a91a-0b6d25268b11"
    }, {
      "connectorId" : "e3306678-a53a-4964-9040-1c96f36dda72"
    } ]
  }
}
2020-11-12 12:24:20.271 PM ERROR io.fusionauth.api.service.system.kickstart.KickstartRunner - Error response:
{
  "fieldErrors" : {
    "tenant.connectorPolicies[1].connectorId" : [ {
      "code" : "[inactiveLicense]tenant.connectorPolicies.connectorId",
      "message" : "You must activate the Reactor using a valid license before using this connector."
    } ],
    "tenant.connectorPolicies[1].domains" : [ {
      "code" : "[empty]tenant.connectorPolicies.domains",
      "message" : "You must specify at least one domain in the [tenant.connectorPolicies[1].domains] property. Specify single entry of '*' to indicate all email domains."
    } ],
    "tenant.connectorPolicies[2]" : [ {
      "code" : "[duplicate]tenant.connectorPolicies",
      "message" : "The [tenant.connectorPolicies] contains more than one policy for the same connector Id. Each connector Id can be specified at most once in a tenant connector policy. The duplicate connector Id is [e3306678-a53a-4964-9040-1c96f36dda72]."
    } ]
  }
}
```

**Comment by mooreds** [comment]
re #2: That sure seems like a bug, since the API docs specify if you don't provide a domain, the default is `[*]`. I'll open a separate bug to track.

re: #3: Patch has some complexities, as documented here: https://github.com/FusionAuth/fusionauth-issues/issues/441 Do the issues appear with a full `PUT`?

**Comment by robotdan** [comment]
re: 1) Reactor License, Kickstart supports adding your license key. #949 is a separate issue to manage license activation via API after the system has been setup via kickstart or other method.

**Comment by ruckc** [comment]
Ok, the reactor license works, it took a while to find how to add the license.

The PUT would be work but it would require significant additional kickstarts for the adminUserFormId and JWT access token key.

I think for now i'll just use the PATCH which appends the ldapConnector.

**Comment by mooreds** [comment]
Ah, so patching just to append works?

In other words, does the below work? 

```
{
            "method": "PATCH",
            "url": "/api/tenant/{{ .Values.auth.defaultTenantId }}",
            "body": {
              "tenant": {
                "connectorPolicies": [
                  {"connectorId": "#{ldapConnector}" }
            
                ]
              }
            }
          },
```

**Comment by ruckc** [comment]
Yes, but would prefer to prepend the connector instead of append it to the list.

**Comment by robotdan** [comment]
Is this still an issue, or is this now possible?
